src/test/kotlin/racingcar/game/RacingGameTest.kt

```diff
@@ -0,0 +1,83 @@
+package racingcar.game
+
+import io.kotest.assertions.throwables.shouldNotThrowAny
+import io.kotest.core.spec.style.DescribeSpec
+import io.kotest.datatest.withData
+import io.kotest.inspectors.forAll
+import io.kotest.matchers.collections.shouldContainExactly
+import io.kotest.matchers.ints.shouldBeInRange
+import racingcar.car.Car
+import racingcar.car.Cars
+
+class RacingGameTest : DescribeSpec({
```

Comment:
I tried DescribeSpec and liked how it can express tests more intuitively with Kotlin DSLs and more detailed descriptions. However, it feels like there is unnecessary indents and too much to read for some simpler tests. Actually, I had seen this document before, and the most eye-catching one was FunSpec, but I wanted to try Kotest's original style which is why I chose ShouldSpec this time.

I think I will stick with FunSpec for next time. I know it will mostly depend on the team convention but just want to know if you have a preference and why maybe. 😆